src/components/tasks/card/index.tsx

```diff
@@ -13,24 +13,31 @@ const moment = require('moment');
 
 type Props = {
   content: task;
+  shouldEdit: boolean;
```

Comment:
how about `isEditable`?

Reply:
Do you want me to change it's name to something else?
I have already used isEditable for the 'edit' variable and isUserAuthorised for the state variable 'role'
It will look like:
shouldEdit= isEditable;
where,
isEditable = !!query.edit && isUserAuthorised;

If you want me to change it's name then I will look for something better

Reply:
isUserAuthorised will tell us if user has admin rights or not and isEditable will only be 'true' if the query parameter will be  'edit=true' and the user who's logged in has admin roles.